langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibArrayTest.java

```diff
@@ -551,4 +551,9 @@ public void testSort9() {
     public void testSort10() {
         BRunUtil.invoke(compileResult, "testSort10");
     }
+
+    @Test
+    public void testReadOnlyArrayFilter() {
+        BRunUtil.invoke(compileResult, "testReadOnlyArrayFilter");
+    }
```

Comment:
Since we are ding the same thing in all the methods, maybe we can use a `DataProvider` to simplify the code